README.md

```diff
@@ -26,6 +32,10 @@ kintone output plugin for Embulk stores app records from kintone.
         - `USER_SELECT`, `ORGANIZATION_SELECT`, `GROUP_SELECT`, `FILE`
     - **timezone**: timezone to convert into `date` (string, default is `UTC`)
     - **val_sep**: Used to specify multiple checkbox values (string, default is `,`)
+    - **sort_columns**: List of columns for sorting rows in SUBTABLE. Available only if type is `SUBTABLE` (array of objects, optional)
+        - **name**: Column name (string, required)
+        - **order**: Sort order (string `asc` or `desc`, required)
+- **chunk_size**: Maximum number of records to request at once (integer, default is `100`)
```

Comment:
To avoid the following error.
https://jp.cybozu.help/k/en/id/040458.html#anchor_filter_records_error